.eslintrc.js

```diff
@@ -74,6 +74,7 @@ module.exports = {
             },
           },
         ],
+        "no-console": "warn",
```

Comment:
```suggestion
        "no-console": ["warn", { allow: ["warn", "error", "info"] }],
```

I'd suggest changing it to this to prevent linting errors on `console.warn`, `console.info` and `console.error`, since those are used legitimately at various points in the codebase.

Reply:
Some people use console.warn/error often when debugging (myself included), so not sure that helps solve the problem of console calls ending up in production by mistake.